docs/landscape.html

```diff
@@ -721,7 +721,8 @@ <h3><a name="Landscape">Landscape</a></h3>
         </ul>
        </li>
       </ul>
-     <li>m3 bits 6..5 : free</li>
+     <li>m3 bit 6 : free</li>
+     <li>m3 bit 5 : House is protected from town/AI removal</li>
```

Comment:
```suggestion
     <li>m3 bit 5 : House is protected from town removal</li>
```

As for the landscape_grid.